grammars/jsdoc.cson

```diff
@@ -410,6 +410,10 @@
           }
         ]
       }
+      {
+        'match': '^\\s*\\*(?!/)'
+        'name': 'comment.block.documentation.jsdoc'
```

Comment:
Wrong scope-name. It should be `punctuation.section.continuation.comment.js`, or at least just `punctuation.section.comment.js`. [See here](https://github.com/atom/language-javascript/blob/ea43f15/grammars/javascript.cson#L1901-L1905).

But this pattern isn't enclosing any content of its own, so it's incorrect to scope it as a comment.